File: C++/Overclocking.cpp
#pragma once
#include <iostream>
using namespace std;

class Overclocking
{
private:
    float maxSpeedGHz;

public:
    Overclocking() {}

    Overclocking(float maxSpeedGHz)
    {
        this->maxSpeedGHz = maxSpeedGHz;
    }

    void setMaxSpeedGHz(float maxSpeedGHz)
    {
        this->maxSpeedGHz = maxSpeedGHz;
    }

    float getMaxSpeedGHz()
    {
        return maxSpeedGHz;
    }

    ~Overclocking() {}
};

File: Python/Overclocking.py
class Overclocking:
    def __init__(self, max_speed_ghz=0.0):
        self._max_speed_ghz = max_speed_ghz

    def set_max_speed_ghz(self, max_speed_ghz):
        self._max_speed_ghz = max_speed_ghz

    def get_max_speed_ghz(self):
        return self._max_speed_ghz